### PR TITLE
Full hostname in oom kill log lines

### PR DESCRIPTION
**Pull request by thebostik**
Write out the value of PAASTA_HOST for an OOM-killed log's value of hostname if we have it.
Depends on https://github.com/Yelp/paasta/pull/2552.

### CODE REVIEW

File: tests/test_oom_logger.py [hunk 1]
```diff
@@ -86,7 +86,11 @@ def sys_stdin_without_process_name():
 def docker_inspect():
     return {
         "Config": {
-            "Env": ["PAASTA_SERVICE=fake_service", "PAASTA_INSTANCE=fake_instance"]
+            "Env": [
+                "PAASTA_SERVICE=fake_service",
+                "PAASTA_INSTANCE=fake_instance",
+                "PAASTA_HOST=dev42-uswest1adevc-64e7207e-f738-4236-b0c5-8f4fc5e0adbd",
```
Comment (drolando) [inline tests/test_oom_logger.py]: Are you sure about this example? I thought the new PAASTA_HOST variable would only have the hostname in it

Comment (EmanekaT) [reply tests/test_oom_logger.py]: I upgraded PAASTA_HOST to use box's hostname

Comment (thebostik) [reply tests/test_oom_logger.py]: Ok, oops that was a misunderstanding of that change, in that case. I am just going to grab `HOSTNAME` from inside the container, which is currently what gets logged in the logs we need to join to. hostname from outside the container is the fallback, which doesn't have the mesos task id information that is logged from the service (to identify which is the OOM'ing instance on a host).

**Comment by solarkennedy** [comment]
@thebostik is this PR still needed?

**Comment by thebostik** [comment]
Thanks for the poke Kyle, just discarding this.

**Pull request closed without merging**